src/openforms/registrations/contrib/zgw_apis/tests/test_zgw_api_config.py

```diff
@@ -544,3 +550,55 @@ def test_get_zgw_config(self, m):
             api_group = plugin.get_zgw_config({})
 
         self.assertEqual(api_group, self.zgw_group1)
+
+
+@disable_2fa
+@temp_private_root()
+class ZGWApiGroupConfigAdminTests(WebTest):
```

Comment:
please move this in a separate test file

otherwise this will conflict hard with #3509